packages/hoppscotch-sh-admin/src/helpers/userManagement.ts

```diff
@@ -0,0 +1,138 @@
+import { useToast } from '~/composables/toast';
+import { getI18n } from '~/modules/i18n';
+import { UserDeletionResult } from './backend/graphql';
+import { ADMIN_CANNOT_BE_DELETED, USER_IS_OWNER } from './errors';
+
+type IndividualActionMetadata = null;
+type BulkActionMetadata = {
+  areMultipleUsersSelected: boolean;
+  deletedIDs: string[];
+};
+type ActionMetadata = IndividualActionMetadata | BulkActionMetadata;
+
+type HandleUserDeletion = {
+  (deletedUsersList: UserDeletionResult[], metadata: ActionMetadata): void;
+};
+
+type ToastMessage = {
+  message: string;
+  state: 'success' | 'error';
+};
+
+const t = getI18n();
+const toast = useToast();
+
+const displayToastMessages = (
+  toastMessages: ToastMessage[],
+  currentIndex: number,
+  isBulkAction: boolean,
+  metadata: ActionMetadata
```

Comment:
do we require this two parameters

Reply:
Good catch, they were unnecessary; removed in https://github.com/hoppscotch/hoppscotch/pull/3937/commits/dee7864a080ac89afd9fad701c48f5af8618b557.